frontend/packages/kubevirt-plugin/src/components/vm.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+
+import {
+  getName,
+  getNamespace,
+  getUid,
```

Comment:
If we want helpers like this, I'd suggest putting it somewhere that can be used outside of kubevirt.

What is the plan for `kubevirt-web-ui-components`?